src/core/database.cpp

```diff
@@ -1194,12 +1194,23 @@ void Database::EndMovieAssetInsert()
 
 void Database::BeginMovieAssetMetadataInsert()
 {
-	BeginBatchInsert("MOVIE_ASSETS_METADATA", 3, "MOVIE_ASSET_ID", "METADATA_SOURCE", "CONTENT_JSON");
-}
-
-void Database::AddNextMovieAssetMetadata(int movie_asset_id,  wxString metadata_type, wxString json)
-{
-	AddToBatchInsert("itt", movie_asset_id, metadata_type.ToUTF8().data(), json.ToUTF8().data());
+	BeginBatchInsert("MOVIE_ASSETS_METADATA", 11, "MOVIE_ASSET_ID", "METADATA_SOURCE", "CONTENT_JSON", "TILT_ANGLE", "STAGE_POSITION_X", "STAGE_POSITION_Y", "STAGE_POSITION_Z", "IMAGE_SHIFT_X", "IMAGE_SHIFT_Y", "EXPOSURE_DOSE", "ACQUISITION_TIME");
```

Comment:
I'm not sure "EXPOSURE_DOSE" makes sense to me. Suggestions from existing fields include TOTAL_EXPOSURE, PRE_EXPOSURE, EXPOSURE_PER_FRAME, EXPOSURE_AMOUNT (though this one is also a bit wonky as the exposure we refer to is actual a number density (e/A^2) and an "amount" should just be a quantity.)

Less opposed to AQUISITION_TIME, however, this could be the time of the aquisition (I assume this is intended as it could be used to sort exposures) or it could read as the duration of each exposure (seconds the beam was on during aquire.) Suggest: TIME_OF_AQUISITION

Reply:
...or TIME_OF_RECORDING or something

Reply:
In other places in cisTEM does TOTAL_EXPOSURE include the PRE_EXPOSURE? Or is it the product of EXPOSURE_PER_FRAME and the number of frames?

Reply:
> 
> 
> In other places in cisTEM does TOTAL_EXPOSURE include the PRE_EXPOSURE? Or is it the product of EXPOSURE_PER_FRAME and the number of frames?

TOTAL_EXPOSURE is used to asses radiation damage in simulate.cpp and reconstruct_3d.cpp, and is used to indicate the exposure the sample has received = pre-expsoure + exposure_per_frame*n_frames. In most cases, this is just pre_exposure_exposure_per_frame as n_frames will be 1 where calculated.